### ISSUE TITLE
--no-stdout-logs is not working, but -n is

### ISSUE THREAD

**Issue by nolanlawson** [pouchdb/pouchdb-server#115]
This seems to be a quirk of how `yargs` deals with the `--no` prefix.

**Comment by marten-de-vries** [comment]
https://www.npmjs.com/package/yargs#negate-fields; probably easiest to just rename the option since I don't see some option to turn it off mentioned.

**Comment by nolanlawson** [comment]
I'm fine with that. Since it's pre-1.0, we can do whatever we want.

How about `--silent` and `-s`?

**Comment by marten-de-vries** [comment]
Sounds good.

**Comment by nolanlawson** [comment]
Well, since we released 1.0.0 and forgot to fix this, I guess we should try to fix it so it actually works?

**Comment by marten-de-vries** [comment]
Absolutely. Fixing issues is more important than keeping semver versions low. PR: #126.

**Comment by nolanlawson** [comment]
This is still broken (`-n` silences successfully, but `--no-stdout-logs` does not), looks like we didn't fix it for some reason. I believe it was discussed but can't recall the resolution.
